lldb/packages/Python/lldbsuite/test/lldbplatformutil.py

```diff
@@ -266,16 +266,11 @@ def getCompiler():
     return module.getCompiler()
 
 
-def getCompilerBinary():
-    """Returns the compiler binary the test suite is running with."""
-    return getCompiler().split()[0]
-
-
 def getCompilerVersion():
     """Returns a string that represents the compiler version.
     Supports: llvm, clang.
     """
-    compiler = getCompilerBinary()
+    compiler = getCompiler()
     version_output = subprocess.check_output([compiler, "--version"], errors="replace")
```

Comment:
Might as well put `getCompiler()` directly in this line, no local variable needed.